packages/neos-ui-containers/src/InsertModeSelector/style.css

```diff
@@ -20,4 +20,9 @@
         border-top-right-radius: 2px;
         border-bottom-right-radius: 2px;
     }
+    .intoButton{
+        margin-left: 8px;
+    }
 }
+
+
```

Comment:
It looks really good already!

The only thing I would add is the distance between the "Above" and the "Below" button. So it would look even, and also a bit prettier.

![SCR-20230225-dtq](https://user-images.githubusercontent.com/39345336/221348646-5ebb096e-4acf-4d2d-b750-dfe7d0605750.png)


In Neos 8.2 it is not yet so completely in the eye stung, because here before only the icon was without text. With text, however, it is a small difference.

![SCR-20230225-dzy](https://user-images.githubusercontent.com/39345336/221348682-9b9562d6-b442-4d05-9f65-7cb49dee08ff.png)

Reply:
I would argue this was done on purpose in the original design from @manuelmeister in https://github.com/neos/neos-ui/issues/3303 and I really like this visual distinction! (And others did. as well in the thread)

Reply:
Yes, I just looked at it. But in my opinion it looks a bit squashed together. Why should I position two buttons next to each other, but separate the third button from the others. Then I'd rather make it uniform with an even spacing.

Reply:
I think the distinction is important as "above" and "below" present oposites and thus belong closly together while "inside" is a third option wich doesnt belong directly to the previous and should be separated.

Maybe we can settle on increasing the x-padding of each button ever so slightly? (but not the gap)

Reply:
> Maybe we can settle on increasing the x-padding of each button ever so slightly?
We can try

Reply:
Yes, the gap is intended, as often the content inside and beside is different but rarely (if ever) different between over and under.

Reply:
I played around locally and I would suggest shrinking the distance between the icon and text and increasing the padding: 
![Bildschirmfoto 2023-03-03 um 11 09 21](https://user-images.githubusercontent.com/91674611/222693275-f96152c6-c220-4a2d-997e-75b7339e5b0b.png)

Does this work for everyone?

Reply:
yes this looks great ;)

Reply:
Looks better.